10_PredictionExamples.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>Is this something you will do or something that might be "fun" to do? This section seems oddly placed and I'm not sure what the purpose is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>